src/Sylius/Bundle/ApiBundle/Command/CommandAwareDataTransformerInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+interface CommandAwareDataTransformerInterface
```

Comment:
I added new tag interface, because some `DataTransformer` can be executed only one in certain object, if we have, for example two `DataTransformers` where `supportsTransformation (...)` equals to us conditions, will be executed first in line .